README.md

```diff
@@ -39,7 +39,7 @@ or
     - Use the --setPassIfNoBaseline if used with snyk prevent commit status and the project is not monitored. This will prevent snyk-prevent_commit_status to fail.
     setPassIfNoBaseline default to false.\
      `snyk test --json --print-deps | snyk-delta --baselineOrg xxx --baselineProject xxx --setPassIfNoBaseline true`
-    - Use the --dontPassOnFail for snyk delta to fail if a new issues are found in a project. Default value: true
+    - Use the --passOnFail for snyk delta to fail if a new issues are found in a project. Default value: false
```

Comment:
this is a breaking change, your commits do not reflect that. We need to release a new major version. Is the breaking change intentional?

Reply:
"passOnFail" and "snyk delta to fail if a new issues are found in a project" do not make sense. passOnFail true means it'll pass if new issues are found, passOnFail false means it'll fail if new issues are found. Is that right?